src/amo/actions/reviews.js

```diff
@@ -124,7 +124,7 @@ export type SetReviewReplyAction = {|
 export const setReviewReply = ({
   originalReviewId,
   reply,
-}: SetReviewReplyParams = {}): SetReviewReplyAction => {
```

Comment:
This issue here, which is the same issue that has caused the majority of the changes to be made for this patch, is that `SetReviewReplyParams` is defined as an exact type, which means that it is an object that must have exact properties, but an empty object (`{}`) doesn't have those properties. For whatever reason, Flow used to not be concerned about this, but is now. One of the solutions to this is to not pass an empty object as a default if it isn't needed. This is the route I have taken in many cases where no default is in fact needed, including this one.